### PR TITLE
Fix vagrant

### PR DESCRIPTION
**Pull request by shaftoe**
**Description:**

I tried https://home-assistant.io/docs/installation/vagrant/ instruction recently and it doesn't work out of the box, apparently one dependency script disappeared from the code base and broke the process. This patch should fix it, plus introduce an easier UX for interacting with the Vagrant box development environment.

### CODE REVIEW

File: source/_components/binary_sensor.insteon_plm.markdown [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+title: "Insteon PLM Binary Sensor"
```
Comment (mnoorenberghe) [inline source/_components/binary_sensor.insteon_plm.markdown]: Hi @shaftoe, you seem to have some unrelated commits in your pull request. Can you please rebase your one commit?

Comment (shaftoe) [inline source/_components/binary_sensor.insteon_plm.markdown]: Ups, will fix

**Comment by shaftoe** [comment]
I want to pull against `current` and not sure how to do from here, will create a new PR, sorry for the mess

**Comment by shaftoe** [comment]
For the records: https://github.com/home-assistant/home-assistant.github.io/pull/2163
